login.html

```diff
@@ -1,11 +1,61 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="ko">
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Document</title>
+    <title>로그인</title>
+    <link
+      rel="stylesheet"
+      as="style"
+      crossorigin
+      href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/variable/pretendardvariable.min.css"
+    />
+    <link rel="stylesheet" href="./styles/reset.css" />
+    <link rel="stylesheet" href="./styles/login_style.css" />
   </head>
   <body>
-    로그인
+    <main>
+      <header>
+        <a href="./index.html">
+          <img src="./img/logo.png" alt="판다마켓 로고" />
+        </a>
+      </header>
+      <form>
+        <label for="email"> 이메일 </label>
+        <input type="email" id="email" />
+        <div class="pw_area">
+          <label for="password"> 비밀번호 </label>
+          <div class="visible_icon"></div>
+          <input type="password" name="" id="password" />
+        </div>
+        <a href="" class="primary_btn">로그인</a>
```

Comment:
❗️ 수정요청
로그인은 페이지 이동이 아니라 추후 특정 이벤트를 실행하는 역할을 하므로 button 태그를 사용하시는 것을 추천드려요!

```suggestion
        <button type="submit" class="primary_btn">로그인</button>
```